Feat/qt

maugustooo opened the pull request:
## Description

added generalInfo and systemInfo code to qt, logo loader and documentation.

---
## Type of Change

- [x] Documentation
- [x] Feature
- [ ] BugFix
- [ ] Other

---

maugustooo committed:
qt improvements

maugustooo committed:
qt improvements

maugustooo committed:
QT info provider

maugustooo committed:
QT info provider

maugustooo committed:
Enhance README with Raspberry Pi deployment details

Added deployment instructions and a script for Raspberry Pi.

maugustooo committed:
update gitignore

rmsa42 reviewed (commented):
Need to put some files in the .gitignore

maugustooo committed:
Merge branch 'feat/QT' of github.com:SEAME-pt/Team2_TheySEAME_Rollin into feat/QT

maugustooo committed:
Revise README for Qt Application on Raspberry Pi 5

Updated the README to enhance formatting and clarity, added weather monitoring feature, and corrected speed measurement unit.

maugustooo committed:
changes readme

maugustooo committed:
Update README.md

maugustooo committed:
Revise README sections for clarity and updates

Updated section headers and removed redundant text.

maugustooo committed:
rename the project files

maugustooo committed:
Merge branch 'feat/QT' of github.com:SEAME-pt/Team2_TheySEAME_Rollin into feat/QT

maugustooo committed:
start CAN comunication

maugustooo committed:
Revise README for hardware, software, and build steps

Updated requirements and compilation instructions in README.

maugustooo committed:
changes on code and add logo loader

maugustooo committed:
Merge branch 'feat/QT' of github.com:SEAME-pt/Team2_TheySEAME_Rollin into feat/QT

maugustooo committed:
Merge branch 'develop' into feat/QT
